### ISSUE TITLE
Vim abbreviations - :ab/:iab

### ISSUE THREAD

**Issue by laballab** [asvetliakov/vscode-neovim#299]
Hi, 

Running into an interesting issue with this, 
:ab/:iab doesn't seem to be working in vscode:1.47.0 even though it works in nvim just fine..

I can confirm my neovim path is set correctly in vscode, other vim features are working as expected.
Doing :ab in nvim launched from the shell works as expected as well..

Could this be a configuration issue or is :ab not supported? 

Appreciate the feedback.

**Comment by laballab** [comment]
I see, thanks for clarifying. Looking thru the src & docs the design choice makes sense now. 

Sounds like any nvim insert-mode plugins are also affected similarly I'm guessing.

**Comment by asvetliakov** [comment]
Correct, anything involving the insert mode on neovim side won't work
